RELEASE-NOTES.txt

```diff
@@ -7,6 +7,7 @@
 - [*] Orders: Fixes order details so separate order items are not combined just because they are the same product or variation. [https://github.com/woocommerce/woocommerce-ios/pull/9710]
 - [Internal] Store creation: starting May 4, store creation used to time out while waiting for the site to be ready (become a Jetpack/Woo site). A workaround was implemented to wait for the site differently. [https://github.com/woocommerce/woocommerce-ios/pull/9731]
 - [**] Mobile Payments: Tap to Pay is initialised on launch or foreground, to speed up payments [https://github.com/woocommerce/woocommerce-ios/pull/9750]
+- [*] Store Creation: Local notifications are used to support users during the store creation process. [https://github.com/woocommerce/woocommerce-ios/pull/9769]
```

Comment:
nit: Maybe we can point to some PRs with testing steps for each scenario?

Reply:
Yup, updated in 2ed1e4fe1ba35d4e9f705d9672ef444a07519eb8.